src/test/java/org/fcrepo/client/TransactionBuilderTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree.
+ */
+package org.fcrepo.client;
+
+import java.net.URI;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * Tests to check the TransactionBuilder constructor works as expected. Method testing done in TransactionBuilderIT.
+ *
+ * @author mikejritter
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class TransactionBuilderTest {
```

Comment:
We should add tests to verify that urls in the form "http://localhost:8080/rest/fcr:tx/fd4f48c4-7c47-46ee-8907-af6411ae4215" work as expected.